css/layout.css

```diff
@@ -500,4 +550,42 @@ summary {
   .form button {
     margin-left: 32%;
   }
+
+  .container-Pop {
+    display: none;
+  }
+
+  .card-pop {
+    display: flex;
+    flex-direction: column;
+    height: 700px;
+    width: 950px;
+    overflow: auto;
+  }

```

Comment:
> This comments refers to line #438

- [ ] I think it would be nice to position the pop-up modal to the center of the window as shown in the design. Currently, the large margin set is causing it to be mis aligned to the right as show below:
![image](https://user-images.githubusercontent.com/61377322/146304794-e7128025-09e1-4adc-a6f3-c670bfcfb546.png)